contexts/did-v1.jsonld

```diff
@@ -41,40 +39,26 @@
       "@id": "sec:controller",
       "@type": "@id"
     },
-    "created": {
-      "@id": "dc:created",
-      "@type": "xsd:dateTime"
-    },
     "blockchainAccountId": "sec:blockchainAccountId",
     "keyAgreement": {
       "@id": "sec:keyAgreementMethod",
       "@type": "@id",
       "@container": "@set"
     },
-    "publicKey": {
-      "@id": "sec:publicKey",
-      "@type": "@id",
-      "@container": "@set"
-    },
     "publicKeyBase58": "sec:publicKeyBase58",
-    "publicKeyPem": "sec:publicKeyPem",
     "publicKeyJwk": {
       "@id": "sec:publicKeyJwk",
       "@type": "@json"
     },
     "service": {
-      "@id": "didv:service",
+      "@id": "https://www.w3.org/TR/did-core/#dfn-services",
```

Comment:
hmm, this looks incorrect. Defining vocabulary IDs in DID Core could be done, but I suggest that we don't want to do that. Let's have a clean vocabulary file... that file is located at: `https://www.w3.org/ns/did` and the representation of that resource is the DID Spec Registries document.

So, the URL should be `https://www.w3.org/ns/did#service`.